src/api/Nhs.Appointments.Api/FunctionConfigurationExtensions.cs

```diff
@@ -65,6 +65,8 @@ public static IFunctionsWorkerApplicationBuilder ConfigureFunctionDependencies(t
             .AddTransient<ISiteStore, SiteStore>()
             .AddTransient<INotificationConfigurationStore, NotificationConfigurationStore>()
             .AddTransient<IAttributeDefinitionsStore, AttributeDefinitionsStore>()
+            .AddTransient<IWellKnownOdsCodesStore, WellKnownOdsCodesStore>()
+            .AddTransient<IWellKnowOdsCodesService, WellKnownOdsCodesService>()
```

Comment:
We have a few of these now, maybe we could have a system document service/store?